src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1095,6 +1099,20 @@ public void decrementSelectedCount()
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void onForwardedEndpointsChanged(
```

Comment:
I think it's unwritten, but we try to use `onXXX` names for "subscription" calls, and just something like `forwardedEndpointsChanged` for when the event occurs.